scenarios/glue_privesc/terraform/rds.tf

```diff
@@ -0,0 +1,80 @@
+resource "aws_db_instance" "cg-rds" {
+  allocated_storage    = 20
+  storage_type         = "gp2"
+  engine               = "postgres"
+  engine_version       = "13.7"
+  instance_class       = "db.t3.micro"
+  db_subnet_group_name = aws_db_subnet_group.cg-rds-subnet-group.id
+  db_name              = var.rds-database-name
+  username             = var.rds_username
+  password             = var.rds_password
+  parameter_group_name = "default.postgres13"
+  publicly_accessible  = false
+  skip_final_snapshot  = true
+
+  port = "5432"
+
+  storage_encrypted = true
+
+  vpc_security_group_ids = [
+    aws_security_group.cg-rds-security-group.id,
+  ]
+
+  depends_on = [local_file.sql_file]
+  tags = {
+    Name     = "cg-rds-instance-${var.cgid}"
+    Stack    = "${var.stack-name}"
+    Scenario = "${var.scenario-name}"
+  }
+}
+
+data "local_file" "csv_file" {
+  filename = "../assets/order_data2.csv"
+}
+
+data "template_file" "sql_template" {
```

Comment:
Switch to use the [templatefile function](https://developer.hashicorp.com/terraform/language/functions/templatefile).

The provider last update was 3 years ago and it does not support Apple M1 macs (me 😢)
> Error: Incompatible provider version
> Provider registry.terraform.io/hashicorp/template v2.2.0 does not have a package available for your current platform, darwin_arm64.

Reply:
I replaced it with using templatefile Fuction! On my PC the data is inserted fine, please let me know if you encounter any errors! :))